src/Plivo/Exceptions/PlivoResponseException.php

```diff
@@ -29,7 +29,7 @@ class PlivoResponseException extends PlivoRestException
      * @param array $responseData
      * @param integer $statusCode
      */
-    public function __construct($message = "", $code = 0, Throwable $previous = null, $responseData, $statusCode)
+    public function __construct(?string $message, ?int $code, $responseData, $statusCode)
```

Comment:
nullable works in php 8
Few question:
1) previously status code is zero , here is nullable
2)previous param is missing

Reply:
@narayana-plivo
1. There is no user-defined exception code, however, we have `$statusCode` param defined to fetch the HTTP status code.
2. The `$previous` param wasn't used anywhere in the code.
Should this be reverted?

Reply:
2. there are 24 reference for PlivoResponseException. Please careful on changing functional params. I have pushed the change and mentioned version too.

Reply:
Sure